doc/support.md

```diff
@@ -32,3 +41,5 @@ Feature releases
     :start-after: <!-- Include start support -->
     :end-before: <!-- Include end support -->
 ```
+
+[1]: Canonical provides additional support through the [Ubuntu Pro](https://ubuntu.com/pro) offering.
```

Comment:
```suggestion
[^1]: Canonical provides additional support through the [Ubuntu Pro](https://ubuntu.com/pro) offering.
```

Reply:
Looks like that didn't work :/

Reply:
The footnote syntax is correct and works in local build (as well as in a couple other pages that are live), but I wonder if possibly the include directive that's between the footnote reference and the corresponding definition is confusing the RTD build.

You might try moving the footnote definition above the include directive, or I can try spinning up a test RTD site in my morning and testing it there, and I can let you know if it makes a difference.

Reply:
I tried it on a test RTD site, and as I suspected, the issue did have to do with the RTD build regarding footnotes and directives.

It's not the include directive (to the README) that's the issue though, the issue seems to be that the comments on lines 4 and 39 (`<!-- Include start release -->` and `<!-- Include end release -->`) delineate a section of Markdown to be inserted via an include directive into another page, `installing.md` (lines 18-21 there). The fix should hopefully be to move the footnote reference so that it's before the `<!-- Include end release -->` comment, which should add it to the included section and keep RTD from complaining that there's a footnote reference without a definition.

I will add it as a suggested change and we can see if that does the trick.

Reply:
Yes, that did work, thanks!